tests/e2e/config/wordpress-debug-log/log-ignore-list.js

```diff
@@ -4,7 +4,7 @@
  * @type {Object} WordPress debug log entries to ignore, keyed by WordPress version.
  */
 export const logIgnoreList = {
-	'5.2.16': [
+	'5.2.21': [
```

Comment:
Necessary to match the version running or jobs will fail to ignore these. We could think about making this looser or allow for a wildcard like `5.2.*` but this is fine for now as it doesn't change often.